### PR TITLE
Adding Viridis Testnet & Mainnet

### PR DESCRIPTION
**Pull request by dwinea**

### CODE REVIEW

**Comment by dwinea** [comment]
I've changed the shortName for testnet to make it unique

**Comment by dwinea** [comment]
I've fixed json prettier format issue in both EIP files and in viridis.json icon file

**Review by ligi — APPROVED** [review]

File: _data/chains/eip155-224.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "decimals": 18
   },
   "infoURL": "https://viridis.network",
```
Comment (Enes9019) [inline _data/chains/eip155-224.json:12]: Fhgv

Comment (Enes9019) [reply _data/chains/eip155-224.json:12]: denes6712@gmail.com

**Pull request merged**